### PR TITLE
br: support autoIncrementID, autoRandomID and sequenceID in PiTR

### PR DESCRIPTION
**Pull request by joccau**
Signed-off-by: joccau <zak.zhao@pingcap.com>

### What problem does this PR solve?

Issue Number: close #xxx

Problem Summary:
- rename the command `br stream` to `br log`.
- support restoring sequenceID/auto_incrementID/auto_randomID.
- report other TiDBs after update global schema version.
- 

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by YuJuncen — APPROVED** [review]

**Review by YuJuncen — CHANGES_REQUESTED** [review]
pending for reviewing the new added parts

**Review by YuJuncen — APPROVED** [review]

File: br/pkg/stream/rewrite_meta_rawkv.go [hunk 1]
```diff
@@ -282,6 +287,48 @@ func (sr *SchemasReplace) rewriteKVEntryForTable(e *kv.Entry, cf string) (*kv.En
 	return &kv.Entry{Key: newKey, Value: newValue}, nil
 }
 
+func (sr *SchemasReplace) rewriteKVEntryForAutoTableIDKey(e *kv.Entry, cf string) (*kv.Entry, error) {
+	newKey, needWrite, _, err := sr.rewriteKeyForTable(
```
Comment (3pointer) [inline br/pkg/stream/rewrite_meta_rawkv.go:291]: needWrite seems always be true, if err is nil？

Comment (joccau) [reply br/pkg/stream/rewrite_meta_rawkv.go:291]: This variable is used to filter table.
If the table need ignore, the needWrite is a false.

**Review by 3pointer — APPROVED** [review]

**Pull request merged**